LiskHQ/lisk-desktop
# Make the line chart of the wallet page generic - closes #2625

### What issue have I solved?

-- #2625

### How have I implemented/fixed it?

- [x] Create a new generic chart component (toolbox/Graphic).
- [x] Added some default settings for the 3 charts that we going to (line, bar and doughnut).
- [x] Added unit test for the graphicOptions.js file
- [x] Replace the existing balance chart graphic component to use the new one (Graphic). 

### How has this been tested?

1. Run the app.
2. Go to Wallet page.
3. Check the balance chart

**NOTE:** I tested the bar and doughnut chart as weell, I made some simple tests to align the settings for those chart same as the design that we have, but was only for set the settings and see if works. The implementation of these other charts will be when we implement the charts for monitor.

### Review checklist
- The PR follows our [Test guide](/LiskHQ/lisk-desktop/blob/development/docs/TEST_GUIDE.md)
- The PR follows our [CSS guide](/LiskHQ/lisk-desktop/blob/development/docs/CSS_GUIDE.md)
